### DISCUSSION TITLE
Can it generate diff between v2 api-docs and v3 api-docs

### DISCUSSION THREAD

**Discussion by ganeshrnayak** [OpenAPITools/openapi-diff#366] [Q&A] [answered]
Hi,

Can this tool be used to generate diff between swagger 2 api-docs and OAS 3 api-docs?

**Comment by joschi** [comment, marked as answer]
Yes, the comparison of Swagger 2 and OAS3 specifications is supported.

See for example this test case:

https://github.com/OpenAPITools/openapi-diff/blob/2.0.1/core/src/test/java/org/openapitools/openapidiff/core/Swagger2CompatibilityTest.java
